scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -281,26 +284,33 @@ def get_threshold_metrics(y_true, y_pred):
 }
 
 
-# In[22]:
-
-def get_coefficients(classifier, X_mat):
-    coef_df = pd.DataFrame.from_items([
-        ('feature', X_mat.columns),
-        ('weight', classifier.coef_[0]),
-    ])
+# In[19]:
 
+def get_coefficients(classifier, feature_set):  
```

Comment:
We could move it. I'm not sure if there is a better way to extract these, or if we even should be doing it this way. The only reason I see for not moving it is if the researchers wants to change the dimensionality reduction, the function as-is will break.

I think what we probably want here is an inverse transformation on the pipeline to find out which genes have the highest loadings on the components that matter?